### ISSUE TITLE
Discord Status Integration Not Working

### ISSUE THREAD

**Issue by NC1107** [ytmdesktop/ytmdesktop#656]
- [X] I understand that **YTMDesktop has NO affiliation with Google or YouTube**.
- [X] I verified that there is no open issue for the same subject.

**Describe the bug**
Discord rich presence integration suddenly is not working while I'm playing music, I'm sure I'm doing something dumb and that's why it's not working but if you can help it's appreciated thanks.

**To Reproduce**
unsure how to reproduce, the discord status normally works fine just as of this morning it has stopped working completely
I'm tried:
-reinstalling
-running discord and/or YTMD as administrator
-adding it to the discord game list but it doesn't appear
-toggling the setting and restarting
-made sure everything was up to date.

**Expected behavior**
I expect to see what music I'm listening to on discord

**Screenshots**

**Environment (please complete the following information):**
 * YTMDesktop version: ***vx.x.x***
 * OS: ***Windows***
 * OS version: ***Windows 10 Latest build***
 * Arch: ***x64*** 
* Installation way: .exe download on site

**Comment by TPRammus** [comment]
Doesn't work for me either. I am a new user though, so it never worked for me.
Win10 x64 with newest Updates. I'm on YTMDesktop v1.14.0, also tried with v1.13.0 though.

- Discord setting is enabled
- Tried re-enabling setting multiple times
- Restarted the application and even my PC
- Tried starting it in administrator mode
- Installed using administrator privileges

**Comment by NC1107** [comment]
i managed to fix it recently,

-first I completely uninstalled youtube music and discord, ended all
processes in task manager
-then go to appdata and remove all files involving YTMA and discord
-restart PC
-install both again
its working for me on YTMA V14 and the normal version of discord

goodluck

**Comment by TPRammus** [comment]
Yeah, had the same problem and managed to somehow fix it. Was probably the upgrade from v1.13.0 to v1.14.0, not entirely sure though
